_test/content_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+describe "Content" do
+
+  BannedCharacters = [
+    "\u201C", # U+201C “
+    "\u201D", # U+201D ”
+    "\u201E", # U+201E „
+    "\u2018", # U+2018 ‘
+    "\u2019", # U+2019 ’
+    "\u201A", # U+201A ‚
+    "\u201B", # U+201B ‛
+  ]
```

Comment:
There are a couple of reasons. They are not ASCII characters and we had cases in the past where Markdown processors or tools we used crashed because they were not able to deal with them by default (especially before Ruby 2.0 when input was not considered UTF-8 by default).

Moreover, these characters have valid ASCII alternatives such as `"` and `'` and most of the time they are injected in posts or articles just because someone is copying and pasting from a Word document. In other words, it never happened to us that the use of these chars was intentional, and their use always leaded to troubles or make it more complex to grep/search patterns in our code.

This test it's a way for us to prevent the use of these chars in favor of the ASCII alternatives.